src/microbe_stage/Patch.cs

```diff
@@ -68,6 +69,8 @@ public Patch(LocalizedString name, int id, Biome biomeTemplate, PatchSnapshot cu
         Name = name;
         ID = id;
         BiomeTemplate = biomeTemplate;
+        BiomeTemplate.Background = currentSnapshot.Background;
+        BiomeTemplate.Sunlight.Colour = currentSnapshot.LightColour;
```

Comment:
Hang on, this cannot be safe, right? This is modifying the Biome class which is a registry type, which is absolutely forbidden from being modified after loading by SimulationParameters.